.github/labeler.yml

```diff
@@ -19,6 +19,7 @@
     - tests/modules/programs/thunderbird/*
     - modules/services/imapnotify*.nix
     - modules/programs/mu.nix
+    - modules/programs/msmtp*.nix
     - modules/programs/lieer.nix
     - modules/services/lieer.nix
     - modules/programs/msmtp*.nix
```

Comment:
just realized it looks like it was already in there

Reply:
shite. follow up at https://github.com/nix-community/home-manager/pull/6883